closed/make/modules/openjceplus/Lib.gmk

```diff
@@ -0,0 +1,64 @@
+# ===========================================================================
+# (c) Copyright IBM Corp. 2023, 2023 All Rights Reserved
+# ===========================================================================
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.
+#
+# IBM designates this particular file as subject to the "Classpath" exception
+# as provided by IBM in the LICENSE file that accompanied this code.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
+# ===========================================================================
+
+include LibCommon.gmk
+
+ifeq (true,$(BUILD_OPENJCEPLUS))
+
+# Identify the desired JGSKIT target platform.
+GSKIT_HOME := $(TOPDIR)/OpenJCEPlus/ock/jgsk_sdk
+JGSKIT_TARGET :=
+JGSKIT_MAKE := jgskit.mak
+JGSKIT_MAKE_PATH := $(OPENJCEPLUS_TOPDIR)/src/main/native/
+JCE_CLASSPATH := $(OPENJCEPLUS_TOPDIR)/src/main/java:$(TOPDIR)/src/java.base/share/classes
```

Comment:
This should refer to where the `java.base` class files are found rather than where (a subset of) the source files live.

Reply:
@keithc-ca Do you know which path variable in the OpenJDK makefile is used for point to the `java.base` class files path?

Reply:
I would start with `$(JDK_OUTPUTDIR)/modules/java.base` (e.g. see `DDR.gmk`), but it might eventually make sense to use `-source-path` and `-module-path`.

Reply:
@keithc-ca 
This `JCE_CLASSPATH` is used when building the OpenJCEPlus native codes. I tried to change it to below.

`JCE_CLASSPATH := $(OPENJCEPLUS_TOPDIR)/target/classes:$(SUPPORT_OUTPUTDIR)/j9jcl/java.base/share/classes`

But get the following error. I think, when building the OpenJCEPlus native codes, the openjceplus java module and the java.base class files are not generated yet. And also, the native codes are not a part of openjceplus.jmod module, So, get this `package jdk.internal.logger is declared in module java.base, which does not export it to the unnamed module` error.

```
Compiling OpenJCEPlus native
/home/tao/mkJDK/openj9-openjdk-jdk17/OpenJCEPlus/src/main/java/com/ibm/misc/Debug.java:15: error: package jdk.internal.logger is not visible
import jdk.internal.logger.SimpleConsoleLogger; 
                   ^
  (package jdk.internal.logger is declared in module java.base, which does not export it to the unnamed module)
/home/tao/mkJDK/openj9-openjdk-jdk17/OpenJCEPlus/src/main/java/com/ibm/misc/Debug.java:16: error: package sun.util.logging is not visible
import sun.util.logging.PlatformLogger;
```